app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -369,15 +370,32 @@ class RepresenteeController @Inject() (
 
         case IncompleteRepresenteeAnswers(Some(name), Some(id), dateOfDeath, Some(contactDetails), true) =>
           val completeAnswers = CompleteRepresenteeAnswers(name, id, dateOfDeath, contactDetails)
-          Ok(cyaPage(completeAnswers, representativeType, journey.isRight))
+
+          updateDraftReturnAndSession(completeAnswers, journey).fold(
+            { e =>
+              logger.warn("Could not update draft return or session", e)
+              errorHandler.errorResult()
+            },
+            _ => Ok(cyaPage(completeAnswers, representativeType, journey.isRight, backLink))
+          )
 
         case c: CompleteRepresenteeAnswers =>
-          Ok(cyaPage(c, representativeType, journey.isRight))
+          Ok(cyaPage(c, representativeType, journey.isRight, backLink))
 
       }
 
     }
   }
+
+  private def backLink(): Call = returns.triage.routes.CommonTriageQuestionsController.whoIsIndividualRepresenting()
```

Comment:
this seems a bit too specific to have in it's own `def` - this is only going to be the back link in a limited number of places. You can just use this value in place - if you want to make it shorter you can just import something at  the top:
```
import returns.triage.{routes => triageRoutes}
```